modules/network-api/src/main/java/org/apache/ignite/internal/network/ClusterService.java

```diff
@@ -57,7 +57,6 @@ public interface ClusterService extends IgniteComponent {
     /** {@inheritDoc} */
     @Override
     default CompletableFuture<Void> stopAsync(ComponentContext componentContext) {
```

Comment:
such default implementation is error-prone. Can we remove the default behavior here?

Reply:
> such default implementation is error-prone.

IMHO, it sounds arguable. On the other hand, the API should be symmetric I think. This means that if we have a default implementation for `stopAsync` then it would be nice to add a default implementation of `startAsync` as well (or remove them both). Anyway, it looks to me that such improvement is out of the scope of the ticket.